test/collectors_test.lua

```diff
@@ -6,29 +6,27 @@ local g = t.group('collectors')
 local metrics = require('metrics')
 local utils = require('test.utils')
 
-g.before_all = function()
+g.before_all( function()
```

Comment:
nil: space here is redundant: `g.before_all( function()` -> `g.before_all(function()`

Reply:
Removed redundant spaces in `before_all`/`after_each` calls.